app/code/Magento/Sales/etc/adminhtml/system.xml

```diff
@@ -49,9 +49,9 @@
                 </field>
             </group>
             <group id="zerograndtotal_creditmemo" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
-                <label>Allow Zero GrandTotal</label>
+                <label>Allow Zero GrandTotal for Credit Memos</label>
```

Comment:
This is not needed - understandable from context

Reply:
I think we should add label - 'Allow Zero GrandTotal for Credit Memos' 
because if we have only label - 'Allow Zero GrandTotoal' - how can one understand about what we are talking? 

It is fine from your point that - I will name filed - 'Allow Zero GrandTotal' - that is understandable

Reply:
@viral-wagento it was just fine: "Allow Zero GrandTotal" for [potential] group of fields, "Allow Zero GrandTotal for Creditmemo" for specific one.

Reply:
@orlangur 
Ohh, I have updated my changes earlier, let me know if any.
Thanks!

Reply:
@viral-wagento to summarize: changes in `app/code/Magento/Sales/etc/adminhtml/system.xml` are not needed, changes `creditmemo` -> `credit memo` are not needed, please squash all needed changes into a single commit and force push.

Reply:
@orlangur Do you think it should be "Allow Zero GrandTotal" or "Allow Zero GrandTotal for Credit Memos" in system xmk? @viral-wagento can you please also remove unused translation from csv, once decided with system.xml

Reply:
@orlangur and @sivaschenko 
once system.xml changes are finalised from both of you, i will do changes in system.xml and its respected changes in translation csv.
Thanks!

Reply:
@sivaschenko should not be changed: "Allow Zero GrandTotal" for group, "Allow Zero GrandTotal for Credit Memos" for field.

Reply:
Done, as per your last comment - @orlangur